### ISSUE TITLE
Keybase will not open to desktop

### ISSUE THREAD

**Issue by GitMarkalsoHub** [keybase/client#25050]
I am getting "starting up keybase" continuously

my log id: 8aea9555fb483812a7400a1c

**Comment by 123Venom** [comment]
Same here. Uninstall/Re-Install did nothing to fix this either.

**Comment by GitMarkalsoHub** [comment]
Yes,  I did the uninstall and reinstall also.  Did the same thing.

**Comment by MaxQuatro** [comment]
Avast reported about keybaserq.exe as FileRepMalware [Trj].

**Comment by 123Venom** [comment]
Something very strange I just noticed.  While on Win10 keybase sits in the `show hidden icons` tray as `Starting up keybase` but if I attempt to launch another instance of `\AppData\Local\Keybase\Gui\Keybase.exe` that then allow the login screen all while that other instance sits with `Starting up keybase`.  Something in 6.0.1 deff messed things up.

**Comment by Mykola-Veryha** [comment]
I just updated to 6.0.2 and now it works well for me

**Comment by GitMarkalsoHub** [comment]
Where is the Windows 6.02 update URL?  I have gone  to the website and I am only seeing 6.02 version for Linux

**Comment by rdavies3** [comment]
Same issue on a mac.  Upgraded this morning, my log ID is 1e7d4435dc894d2e7c84061c

**Comment by 123Venom** [comment]
Lot of issues being reported of having this problem.  Be nice to see it fixed.

**Comment by 123Venom** [comment]
2 weeks later still not fixed nor a response on the matter.

**Comment by 123Venom** [comment]
That's all fine if you are setup to start kb on windows start, I was not.  I tested this by closing kb.  Attempted a manual run with the short name version of the user.  It still does the same thing.  The first instance is blank and sits forever on `starting up keybase`.  If I open a 2nd instance that one then works but that whacky tray instance must remain.

**Comment by ec-max** [comment]
Sorry for the false hope.  You describe the exact same behavior I was seeing on a fresh install (new machine):  First run whacky tray instance wasn't working, second copy worked fine.  Even installed the KBFS support and that mounted up fine with my account/data mounted as a drive letter; but not the whacky first instance tray copy; until I eliminated spaces from the path.   I haven't had to go experiment with this; but looking at the client source files, it is reading from %LOCALAPPDATA% - which has the space included long file name in the environment variable.  If you are still having the issue you may get somewhere by setting that variable to the short filename too.  Worth 5 minutes to try anyway.  ?shrug?
